src/BotPlutusInterface/Config.hs

```diff
@@ -75,6 +81,29 @@ logLevelSpec =
     <!> Info <$ atomSpec "info"
     <!> Debug <$ atomSpec "debug"
 
+instance ToValue TxStatusPolling where
+  toValue (TxStatusPolling interval timeout) =
+    Sections
+      ()
+      [ Section () "pollingInterval" $ toValue interval
+      , Section () "pollingTimeout" $ toValue timeout
+      ]
+
+txStatusPollingSpec :: ValueSpec TxStatusPolling
+txStatusPollingSpec =
+  sectionsSpec "TxStatusPolling configuration" $ do
+    spInterval <-
+      reqSection'
+        "milliseconds"
```

Comment:
This should be microseconds I think, it will use https://github.com/mlabs-haskell/bot-plutus-interface/blob/master/src/BotPlutusInterface/Effects.hs#L156 which takes microseconds as an arg

Reply:
Indeed, thanks! I've changed doc it type itself, but forgot to do it here.